exercises/saddle-points/Cargo.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+name = "saddle-points"
+version = "1.1.0"
```

Comment:
This version should match the version in [`canonical-data.json`](https://github.com/exercism/problem-specifications/blob/master/exercises/saddle-points/canonical-data.json#L3). That means that as of the current `master` branch of `problem-specifications`, it should be `1.0.0`.